src/api/controllers/members.ts

```diff
@@ -80,6 +80,11 @@ export async function handleInternalGuildMemberUpdate(data: DiscordPayload) {
       guildMember?.nick,
     );
   }
+
+  if (guildMember?.pending === false && cachedMember?.pending) {
```

Comment:
```suggestion
  if (payload.pending === false && guildMember?.pending === true) {
```

Reply:
Yes that's better but the `=== true` is not necessary I think.